src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+  height: 100%;
+
+  &__star {
+    height: 16px;
+    width: 19.2px;
```

Comment:
strange number, set 16px as well

add margin to each star (except the last one)

Reply:
I have flex for this, watch layout_card

Reply:
yeah, but what we have:

1. Some random number for width, with flex that would be not clear how to change (size or margins between stars) if needed
vs
2. Clear and understandable number for size and margin

Reply:
size is 20% from 166px, and flex is space between all is good

Reply:
in the layout_card everything was with flex, there is no need to change it

Reply:
I agree with the previous mentor. The card width could be changed and to calculate it for every container it's not a best practice. If you want to get it calculated you can use ```calc()``` for dividing the 100% width of the parent container by 5. Now this value seems unclear and while you are looking through the code, you have no idea of how it was calculated.